sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -59,6 +71,163 @@ def get_service_statistics(self, **kwargs):
         result = self._client.get_service_statistics(**kwargs)
         return result.as_dict()
 
+    ### Index Operations
+
+    @distributed_trace
+    def list_indexes(self, **kwargs):
+        # type: (**Any) -> List[Index]
+        """List the indexes in an Azure Search service.
+
+        :return: List of indexes
+        :rtype: list[~azure.search.documents.Index]
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexes.list(**kwargs)
+        return result.indexes
+
+    @distributed_trace
+    def get_index(self, index_name, **kwargs):
+        # type: (str, **Any) -> Index
+        """
+
+        :param index_name: The name of the index to retrieve.
+        :type index_name: str
+        :return: Index object
+        :rtype: ~azure.search.documents.Index
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexes.get(index_name, **kwargs)
+        return result
+
+    @distributed_trace
+    def get_index_statistics(self, index_name, **kwargs):
+        # type: (str, **Any) -> dict
+        """Returns statistics for the given index, including a document count
+        and storage usage.
+
+        :param index_name: The name of the index to retrieve.
+        :type index_name: str
+        :return: Statistics for the given index, including a document count and storage usage.
+        :rtype: ~azure.search.documents.GetIndexStatisticsResult
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexes.get_statistics(index_name, **kwargs)
+        return result.as_dict()
+
+    @distributed_trace
+    def delete_index(self, index_name, **kwargs):
+        # type: (str, **Any) -> None
+        """Deletes a search index and all the documents it contains.
```

Comment:
Probably doc only: what's the behavior if the index does not exist? Exception? Nothing?